internal/course/domain/course.go

```diff
@@ -6,18 +6,15 @@ import (
 	"github.com/google/uuid"
 )
 
-// Course struct.
 type Course struct {
-	ID          uint       `json:"id"`
-	UUID        uuid.UUID  `json:"uuid"`
-	Code        string     `json:"code"`
-	Name        string     `json:"name"`
-	Underline   string     `json:"underline"`
-	Image       string     `json:"image"`
-	ImageCover  string     `db:"image_cover"   json:"image_cover"`
-	Excerpt     string     `json:"excerpt"`
-	Description string     `json:"description"`
-	CreatedAt   time.Time  `db:"created_at"    json:"created_at"`
-	UpdatedAt   time.Time  `db:"updated_at"    json:"updated_at"`
-	DeletedAt   *time.Time `db:"deleted_at"    json:"deleted_at"`
+	UUID        uuid.UUID `json:"uuid"`
+	Code        string    `json:"code"`
+	Name        string    `json:"name"`
+	Underline   string    `json:"underline"`
+	Image       string    `json:"image"`
```

Comment:
The fields: `Image`, `ImageCover`, and `Description` are nullable in the database, in this case, we need to add the `omitempty` to the JSON annotation.

Reply:
https://github.com/sumelms/microservice-course/pull/19/commits/6244cbf2156cfdfa417826cf5017f6c73799dd89